devel_requirements.txt

```diff
@@ -0,0 +1,8 @@
+black
+boto3
```

Comment:
nit: you can remove `boto3`, `botocore`, and `pyyaml` and just use `nypl-py-utils[avro-client,config-helper]==1.2.0` instead